### PR TITLE
fix #77946 amongusavatarmaker.com

### LINKED ISSUE #77946: www.amongusavatarmaker.com
### Issue URL (Ads)
 [https://www.amongusavatarmaker.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.amongusavatarmaker.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Firefox
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>EasyList<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Language-specific:</b><br/> EasyList Italy, <br/>Xfiles
Other extensions used: | Norton safeweb

### PR DESCRIPTION
**Pull request by wout-codes**
#77946

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -258,6 +258,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 ||mypornstarbook.net/ads/
+amongusavatarmaker.com##h1 > div[align^="center"] > a[href] > img[border="0"]
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/specific.txt:261]: `||i.ibb.co^$domain=amongusavatarmaker.com`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:261]: I don't trust it, they may change it.

**Review by zloyden — CHANGES_REQUESTED** [review]
I'd add `$image` - `||i.ibb.co^$image,domain=amongusavatarmaker.com`.


> I don't trust it, they may change it.

They can change everything, so, blocking request is ok here.

**Comment by wout-codes** [comment]
All my hard-work in specifying the rule, became useless. I see.

**Review by zloyden — APPROVED** [review]

**Pull request merged**